locales-pending/premium.ftl

```diff
@@ -13,6 +13,8 @@ main-nav-link-dashboard-label = Dashboard
 main-nav-link-faq-label = FAQs
 main-nav-link-faq-tooltip = Frequently asked questions
 
+mobile-menu-label = Main menu
```

Comment:
I added this because the a11y test now caught that the mobile menu did not have a unique label.